### PR TITLE
Adding no_state_change action to default workflow

### LINKED ISSUE #244: Collect new and returning users per week from Google Analytics
Display these numbers as a bar chart in the "User activity" section of the "activity summary"

See projecthydra-labs/hyku#431 for mockup.

### PR DESCRIPTION
**Pull request by jeremyf**
For states in which the role can make a state change, add a new action
that allows is non-state changing.

Closes #244

### CODE REVIEW

**Comment by mjgiarlo** [comment]
Since #224 is about letting users leave comments without changing states, is this sufficient to close the ticket? I'm wondering specifically how this will look in the UI, whether this change communicates to users that it's a mechanism for leaving a comment independent of a state change.

**Comment by jeremyf** [comment]
@mjgiarlo The state could be "leave_a_comment" instead

**Comment by mjgiarlo** [comment]
@jeremyf That might be clearer for users, so :+1:.

File: lib/generators/hyrax/templates/mediated_deposit_workflow.json.erb [hunk 1]
```diff
@@ -58,6 +58,12 @@
                             "to": ["approving"]
                         }
                     ]
+                }, {
+                    "name": "leave_a_comment",
```
Comment (jcoyne) [inline lib/generators/hyrax/templates/mediated_deposit_workflow.json.erb]: Are there any states that come from this state?  I'm not sure why this should be a terminal state?

Comment (mjgiarlo) [inline lib/generators/hyrax/templates/mediated_deposit_workflow.json.erb]: This is a new action, right, not a state? The way I read this change is that `leave_a_comment` is a new action that can be triggered from three current states, and the new action doesn't trigger a state change. Have you tested this in the UI to ensure it does what you expect, @jeremyf?

Comment (jeremyf) [inline lib/generators/hyrax/templates/mediated_deposit_workflow.json.erb]: @mjgiarlo You read the data correctly. I can write a feature test for this particular behavior if that would be more appropriate (I shudder at UI tests for data driven behavior as its verifying a certain moment, but not verifiable over the course of the project).

**Comment by jcoyne** [comment]
@jeremyf I wonder if these new failures are due to ActiveFedora 11.1.0.  There was some work done around the logging implementation in that release.

**Comment by jeremyf** [comment]
@jcoyne Perhaps :(

**Pull request merged**